src/main/scala/edu/ie3/simona/agent/participant/fixedfeedin/FixedFeedInAgentFundamentals.scala

```diff
@@ -132,10 +132,10 @@ protected trait FixedFeedInAgentFundamentals
     val lastTickInSimulation = simulationEndDate.toTick(simulationStartDate)
     val dataTicks =
       /* As participant agents always return their last known operation point on request, it is sufficient
-       * to let a fixed fixed in model determine it's operation point on:
+       * to let a fixed model determine its operation point on:
```

Comment:
FixedFeedIn model?

Reply:
Oh yes, that could be true. Will change it